core/tests/test_models.py

```diff
@@ -228,3 +229,15 @@ def test_get_saved_posts(user: User, post: Post) -> None:
     assert saved_posts.first().post == post
     SavedPost.remove_saved_post(user=user, post=post)
     assert not saved_posts.exists()
+
+
+@pytest.mark.django_db()
+def test_create_community() -> None:
+    user = User.objects.create_user(
+        email="testuser@example.com", nickname="Test Nickname", password=os.getenv("TEST_PASSWORD")

```

Comment:
you could use `user` fixture conftest.py - password would be generated on the fly

Reply:
Thanks, I have changed this in the code. Good to know!